doc/scrapper/ParsingFiles.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Assuming you have already extracted the files
+
+# In[1]:
+
+
+import re
+#number of entries in this zip: 996181
+filename= "s2-corpus-000"
+filedir= "/home/ricostynha/Desktop/Erasmus/Apps/FoolingAround"
```

Comment:
Can you please replace this path by  `./examples `? This will make the folder `doc/scrapper/example` the directory, independently of your local machine.